### PR TITLE
Remove receipting from consumer service

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
https://trello.com/c/3U4nBGn2/708-708-two-receipts-to-rrm-being-sent-for-each-seft

RAS now handles the receipting when it receives the seft survey making it unnecessary to do it here.  This PR removes the receipting step and goes directly to the virus scanning

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: app/tests/test_consumer.py [hunk 1]
```diff
@@ -50,17 +45,15 @@ def test_valid_message_writes_to_log_after_ftp(self, mock_deliver_binary, mock_s
...
     @staticmethod
     def _contains_statement_in_log_file(statement, output):
         return [statement for line in output if statement in line]
 
     @patch('app.anti_virus_check.AntiVirusCheck.send_for_av_scan')
-    @patch('app.main.SeftConsumer._send_receipt')
     @patch('app.sdxftp.SDXFTP.deliver_binary')
-    def test_valid_message_receipt_sent(self, mock_deliver_binary, mock_send_receipt, mock_send_for_av_scan):
+    def test_valid_message_receipt_sent(self, mock_deliver_binary, mock_send_for_av_scan):
         """Validate that the receipt was sent"""
```
Comment (AndrewTorrance) [inline app/tests/test_consumer.py]: This comment is now out of date

File: app/tests/test_consumer.py [hunk 2]
```diff
@@ -50,17 +45,15 @@ def test_valid_message_writes_to_log_after_ftp(self, mock_deliver_binary, mock_s
...
 
     @staticmethod
     def _contains_statement_in_log_file(statement, output):
         return [statement for line in output if statement in line]
 
     @patch('app.anti_virus_check.AntiVirusCheck.send_for_av_scan')
-    @patch('app.main.SeftConsumer._send_receipt')
     @patch('app.sdxftp.SDXFTP.deliver_binary')
-    def test_valid_message_receipt_sent(self, mock_deliver_binary, mock_send_receipt, mock_send_for_av_scan):
+    def test_valid_message_receipt_sent(self, mock_deliver_binary, mock_send_for_av_scan):
```
Comment (AndrewTorrance) [inline app/tests/test_consumer.py]: Test name now misleading . Something along the lines of test_file_sent_to_av_and_ftp ?may be more appropriate ?

File: app/tests/test_consumer.py (hunk 1 above)
Comment (insacuri) [reply app/tests/test_consumer.py]: Fixed

File: app/tests/test_consumer.py (hunk 2 above)
Comment (insacuri) [reply app/tests/test_consumer.py]: Fixed

**Pull request merged**